dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic

```diff
@@ -19,19 +19,17 @@ StaticConnect                 =             connecting %1%_%2% with %3%
 DynamicConnect                =             connecting %1%_%2% with %3%_%4%
 ModelName                     =             model :
 ConnectedModels               =             model %1% to model %2%
-SolveParameters               =             solve Parameters of model : %1%
+SolveParameters               =             starting local initialization of model %1%
+SolveParametersOK             =             local initialization of model %1% ended successfully
```

Comment:
Yes!

Reply:
next MR will be about the general log, this one is only the beginning ;)